pkg/controller/template/render.go

```diff
@@ -180,6 +180,11 @@ func generateMachineConfigForName(config *RenderConfig, role, name, templateDir,
 	platformDirs := []string{}
 	// Loop over templates/common which applies everywhere
 	for _, dir := range []string{platformBase, platform} {
+		// Bypass OpenStack template rendering until
+		// https://github.com/openshift/installer/pull/1959 merges
```

Comment:
Hum...so if I understand this correctly, this PR is adding all of the code, and then we need to get the installer PR in first, then we'll do a PR to drop these conditionals?

Reply:
Yes that's the intention. The installer PR depends on the changes there, but this PR depends on the changes in the installer PR so this let's us break that cycle.

The conditionals are isolated in a single commit we can revert afterwards.